src/Microsoft.Health.Fhir.Core/Features/Search/Registry/SearchParameterStatusManager.cs

```diff
@@ -88,6 +88,9 @@ public async Task EnsureInitialized()
                 }
             }
 
+            var searchParameterHashValue = SearchHelperUtilities.CalculateSearchParameterHash(parameters.Values);
```

Comment:
I am not familiar with this class/code, but seems like `parameters` can contain search params that are `Disabled`. Do we want to include those also in the has calculation? And do we need to consider values present in `updated`?

Reply:
This is a good point.  I am not sure I have the correct answer right now :).  I created a new work item: 75058, to track this and update it.